dojoWordle/src/components/Game.tsx

```diff
@@ -2,19 +2,26 @@ import styled from '@emotion/native';
 import React, { useState } from 'react';
 import { WordleKeyboard } from './Keyboard';
 import { LetterBox, PossibleLetter, Validity } from './LetterBox';
+import { useKeyboard } from './useKeyboard';
+
+export type Word = PossibleLetter[];
 
 export function Game() {
-  const [letter, setLetter] = useState<PossibleLetter>(null);
+  const { word, addToWord, removeFromWord } = useKeyboard();
 
   return (
     <GameView>
-      <LetterBox letter={letter} validity={checkValidity(letter)} />
+      <WordleLetters testID="wordleLetters">
+        {word.map((letter, index) => (
+          <LetterBox letter={letter} validity={'nofill'} key={index} />
+        ))}
+      </WordleLetters>
       <WordleKeyboard
         onKeyPress={(letter) => {
-          setLetter(letter);
+          addToWord(letter);
```

Comment:
```suggestion
          onKeyPress={addToWord}
```
Ca evite de redéfinir une fonction anonyme à chaque rendu